### PR TITLE
Add GetBlock to External Services

### PR DESCRIPTION
**Pull request by wetryingtodorock**

### CODE REVIEW

**Comment by wetryingtodorock** [comment]
Can you, please, let me know, when the pull request can go through?

File: docs/Node Operators/run-an-ethereum-client.md [hunk 1]
```diff
@@ -105,6 +105,29 @@ Return to [Running a Chainlink Node](../running-a-chainlink-node/).
...
+```
+```text Goerli
+ETH_URL=wss://eth.getblock.io/goerli/
+```
+```text Sepolia
+ETH_URL=wss://eth.getblock.io/sepolia/
+```
+```text Mainnet
+ETH_URL=wwss://eth.getblock.io/mainnet/
+```
```
Comment (dwightjl) [inline docs/Node Operators/run-an-ethereum-client.md:129]: API keys are necessary, correct?

Comment (wetryingtodorock) [reply docs/Node Operators/run-an-ethereum-client.md:129]: Yes, API keys are required.

**Comment by wetryingtodorock** [comment]
Can you, please, update me here?

File: docs/Node Operators/run-an-ethereum-client.md (hunk 1 above)
Comment (dwightjl) [reply docs/Node Operators/run-an-ethereum-client.md]: Added API key placeholder.

**Review by dwightjl — APPROVED** [review]

**Comment by dwightjl** [comment]
This is now approved. I made a few fixes and reordered this list. Merging now.

**Comment by wetryingtodorock** [comment]
> This is now approved. I made a few fixes and reordered this list. Merging now.
> 
> Thank you for adding this!

Thank you so much!

**Comment by wetryingtodorock** [comment]
> This is now approved. I made a few fixes and reordered this list. Merging now.
> 
> Thank you for adding this!

Hello!
Just found out that the links are not correct, they work only for the dedicated nodes, but not for our shared nodes.

The correct links for the shared nodes:

ETH_URL=wss://eth.getblock.io/rinkeby/?api_key=YOUR_API_KEY
ETH_URL=wss://eth.getblock.io/kovan/?api_key=YOUR_API_KEY
ETH_URL=wss://eth.getblock.io/ropsten/?api_key=YOUR_API_KEY
ETH_URL=wss://eth.getblock.io/goerli/?api_key=YOUR_API_KEY
ETH_URL=wss://eth.getblock.io/sepolia/?api_key=YOUR_API_KEY
ETH_URL=wss://eth.getblock.io/mainnet/?api_key=YOUR_API_KEY

**Comment by dwightjl** [comment]
> > This is now approved. I made a few fixes and reordered this list. Merging now.
> > Thank you for adding this!
> 
> Hello! Just found out that the links are not correct, they work only for the dedicated nodes, but not for our shared nodes.
> 
> The correct links for the shared nodes:
> 
> ETH_URL=wss://eth.getblock.io/rinkeby/?api_key=YOUR_API_KEY ETH_URL=wss://eth.getblock.io/kovan/?api_key=YOUR_API_KEY ETH_URL=wss://eth.getblock.io/ropsten/?api_key=YOUR_API_KEY ETH_URL=wss://eth.getblock.io/goerli/?api_key=YOUR_API_KEY ETH_URL=wss://eth.getblock.io/sepolia/?api_key=YOUR_API_KEY ETH_URL=wss://eth.getblock.io/mainnet/?api_key=YOUR_API_KEY

I made a PR with a fix here: https://github.com/smartcontractkit/documentation/pull/649

Thank you for the followup!

**Comment by wetryingtodorock** [comment]
> #649

Great! Thank you so much!

**Pull request merged**